pages/api/nurses/index.js

```diff
@@ -6,6 +6,18 @@ export default async function handler(request, response) {
 
   if (request.method === "GET") {
     const nurses = await Nurse.find();
-    response.status(200).json(nurses);
+    return response.status(200).json(nurses);
+  }
+  if (request.method === "POST") {
+    try {
+      console.log("Post request to add new nurse:", request.body);
+      const nurseData = request.body;
+      await Nurse.create(nurseData);
+
+      response.status(201).json({ message: "Nurse added!🍻" });
```

Comment:
Likewise, send back the created resource - the success is assumed from the `20x` code ^^

Reply:
Thank you very much for your review and help. I didn't fix everything you suggested just now, but I will do so throughout the project.